scripts/make.py

```diff
@@ -534,6 +562,17 @@ def compiler_str(bin_name: str) -> str:
         if cfg.is_cross_compiling:
             self._copy_runtime_to_native(lib_spec)
 
+        # For newlib, the --with-target-subdir configure option is passed by default
```

Comment:
I think it would be better when building the newlib files to output them in the build directory. as it is the files in 'target' are still there. and in general that gives us a bit more control over what actually ends up in the install directory.